pangeo_forge_recipes/transforms.py

```diff
@@ -547,6 +547,7 @@ def expand(
                 | beam.Map(self.dynamic_chunking_fn, **self.dynamic_chunking_fn_kwargs)
             )
         )
+        logger.warn(f"Storing Zarr with {target_chunks =} to {self.get_full_target()}")
```

Comment:
```suggestion
        logger.info(f"Storing Zarr with {target_chunks =} to {self.get_full_target()}")
```
Seems like this is `info`, since nothing potentially adverse (i.e. warn-worthy) is happening?